composer.json

```diff
@@ -13,6 +13,10 @@
             "email": "henriquemoody@gmail.com"
         }
     ],
+    "require": {
+        "php": ">=5.4.0",
+        "arara/process": "~1.9.1"
+    },
```

Comment:
It must be required only if you want to use this feature. Put `arara/process` on `require-dev` and on `suggest` keys.